superset/contrib/docker/docker-init.sh

```diff
@@ -17,23 +17,27 @@
 #
 set -ex
 
-# Create an admin user
-fabmanager create-admin \
-    --app superset \
-    --username $ADMIN_LOGIN \
-    --firstname $ADMIN_FIRST_NAME \
-    --lastname $ADMIN_LAST_NAME \
-    --email $ADMIN_EMAIL \
-    --password $ADMIN_PASSWORD
-
-# Initialize the database
+# always run migrations
 superset db upgrade
```

Comment:
did you test it?
I see the order of `create-admin` and `db upgrade` are different. Maybe it doesn't actually matter but better check.

Reply:
Tested it, works fine